crates/ra_syntax/src/ast/generated.rs

```diff
@@ -217,7 +217,15 @@ impl<R: TreeRoot<RaTypes>> BinExprNode<R> {
 }
 
 
-impl<'a> BinExpr<'a> {}
+impl<'a> BinExpr<'a> {
+    pub fn lhs(self) -> Option<Expr<'a>> {
+        super::child_opt(self)
+    }
+
+    pub fn rhs(self) -> Option<Expr<'a>> {
+        super::child_opt(self)
```

Comment:
this doesn't really return the RHS, since it'll just return the first child expression :smile: check out how e.g. `ImplBlock` implements `target_type` and `target_trait`.